tests/parametric/test_telemetry.py

```diff
@@ -254,3 +254,77 @@ def test_telemetry_event_not_propagated(self, library_env, test_agent, test_libr
                 assert (
                     "install_signature" not in body["payload"]
                 ), "The install signature should not be included in the telemetry event, got {}".format(body)
+
+
+@rfc("https://docs.google.com/document/d/1xTLC3UEGNooZS0YOYp3swMlAhtvVn1aa639TGxHHYvg/edit")
+@scenarios.parametric
+@features.telemetry_app_started_event
+class Test_TelemetrySCAEnvVar:
+    """
+    This telemetry entry has the value of DD_APPSEC_SCA_ENABLED in the library.
+    """
+
+    @staticmethod
+    def get_app_started_configuration_by_name(library_env, test_agent, test_library):
+        with test_library.start_span("first_span"):
+            pass
+
+        test_agent.wait_for_telemetry_event("app-started")
+        requests = test_agent.raw_telemetry(clear=True)
+        assert len(requests) > 0, "There should be at least one telemetry event (app-started)"
+        for req in requests:
+            body = json.loads(base64.b64decode(req["body"]))
+            if body["request_type"] != "app-started":
+                continue
+
+            assert (
+                "configuration" in body["payload"]
+            ), "The configuration should be included in the telemetry event, got {}".format(body)
+
+            configuration = body["payload"]["configuration"]
+
+            configuration_by_name = {item["name"]: item for item in configuration}
+            return configuration_by_name
+
+        return None
+
+    @staticmethod
+    def get_dd_appsec_sca_enabled_str(library):
+        DD_APPSEC_SCA_ENABLED = "DD_APPSEC_SCA_ENABLED"
+        if library in ("dotnet", "java", "nodejs"):
+            DD_APPSEC_SCA_ENABLED = "appsec.sca.enabled"
```

Comment:
This one seems to be missing at https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json#L30

Reply:
Here it is: https://github.com/DataDog/dd-go/pull/130968